javascript課題①-3.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+
+    const tasks = ["掃除","買い物","散歩"];
+
+    function tasklist() {
```

Comment:
せっかくなので、アロー関数で記述した方が良いと思います！

```suggestion
    const tasklist = () =>
```